.github/workflows/test_pr.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches:
       - main
-  pull_request_target:
+  pull_request:
```

Comment:
It's strange that `pull_request_target` is still running. I assume this is expected? Do we typically revert this line before squashing the PR?

Reply:
Yeah, `pull_request_target` will still run whatever workflow file was committed to `main`.